mender-auth/ipc/server.hpp

```diff
@@ -98,6 +99,7 @@ class Caching {
 	http::Client client_;
 	http::ServerConfig server_config_;
 	http::Server server_;
+	string default_identity_script_path;
```

Comment:
missing `_` at the end of the name

Reply:
Discussed, ignored for readability this time around.